documentation/concepts.md

```diff
@@ -0,0 +1,116 @@
+---
+layout: page
+title: "Concepts"
+--- 
+
+## Introduction
+
+firewalld aims to be a simple, zoning, stateful firewall. Firewall rules
+are organized into zones and policies. Zones logically separate the
+network. Policies apply rules to traffic flowing between zones.
```

Comment:
```suggestion
Firewalld is a simple, stateful, zoned based firewall with a number of features, including policies. Policies and zones are used to organize firewall rules. The network is logically divided into zones and the traffic flowing between them can be governed by policies.
```

Reply:
I added a suggestion to make the introductory summary a little less terse.

Reply:
Good suggestion. I dropped part of the first sentence.

```
Firewalld is a simple, stateful, zoned based firewall. Policies and
zones are used to organize firewall rules. The network is logically
divided into zones and the traffic flowing between them can be governed
by policies.
```